js_modules/dagit/packages/core/src/instigation/InstigationTick.tsx

```diff
@@ -223,6 +223,14 @@ export const TICK_TAG_FRAGMENT = gql`
   ${PYTHON_ERROR_FRAGMENT}
 `;
 
+export const DYNAMIC_PARTITION_REQUESTS_FRAGMENT = gql`
```

Comment:
Technically I feel like this fragment should be colocated with the `DynamicPartitionRequests` component which utilizes the fragment type. Want to move it there maybe?